source/kernel/src/lib.rs

```diff
@@ -202,7 +202,8 @@ impl Kernel {
     }
 
     pub fn spawn_allocated<F: Future + 'static>(&'static self, task: HeapBox<Task<F>>) {
-        self.inner.scheduler.spawn_allocated::<F, HBStorage>(task)
+        // TODO: Join Handle support!
+        let _ = self.inner.scheduler.spawn_allocated::<F, HBStorage>(task);
```

Comment:
i believe the TODO is as simple as changing this to:
```rust
    pub fn spawn_allocated<F: Future + 'static>(&'static self, task: HeapBox<Task<F>>) -> JoinHandle<F> {
        self.inner.scheduler.spawn_allocated::<F, HBStorage>(task)
    }
```
(and an appropriate import somewhere)

Reply:
Yeah. that makes sense! I was just fast-forwarding the dep to get the waitcell fix :D